### DISCUSSION TITLE
Nuget package working on GH Enterprise

### DISCUSSION THREAD

**Discussion by ghost** [community/community#26554] [Enterprise]
<p>I’m having a hard time publishing a NuGet package to Nuget to GH Enterprise. It returns not found for all the URL’s. I already created a token with permission to read/write packages, permission for repo. All packages registry are enabled on Management Console. SSL setup for the nuget.pkg subdomain.</p>
<p>I also added these URLs as sources when trying to push the packages.</p>
<p>Are there any additional settings to have packages working on GH Enterprise that we do not find on the documentation?</p>
<p>Not found:<br>
<a href="https://GHENTERPRISEDOMAIN/_registry/nuget/OWNER/index.json" class="onebox" target="_blank" rel="noopener nofollow ugc">https://GHENTERPRISEDOMAIN/_registry/nuget/OWNER/index.json</a></p>
<p>Not found:<br>
<a href="https://nuget.pkg.GHENTERPRISEDOMAIN/OWNER/index.json" class="onebox" target="_blank" rel="noopener nofollow ugc">https://nuget.pkg.GHENTERPRISEDOMAIN/OWNER/index.json</a></p>
<p>Thanks</p>

**Comment by toddself** [comment, marked as answer]
<p>If subdomain isolation is enabled, the URL will be <code>nuget.GHENTERPRISEDOMAIN/OWNER/index.json</code></p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/enterprise-server@2.22/packages/guides/configuring-dotnet-cli-for-use-with-github-packages" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/enterprise-server@2.22/packages/guides/configuring-dotnet-cli-for-use-with-github-packages" target="_blank" rel="noopener nofollow ugc">Configuring dotnet CLI for use with GitHub Packages - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/enterprise-server@2.22/packages/guides/configuring-dotnet-cli-for-use-with-github-packages" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/enterprise-server@2.22/packages/guides/configuring-dotnet-cli-for-use-with-github-packages</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>(This does not change for version 3.x)</p>

**Comment by ghost** [comment]
<p>Hi <a class="mention" href="/toddself">@toddself</a>, Thank you for your help. It is working now for me with the URL <a href="https://nuget.myghurl/organization/index.json" rel="noopener nofollow ugc">https://nuget.myghurl/organization/index.json</a>. I also have to set up an SSL for this subdomain.</p>
